### DISCUSSION TITLE
yarn and npm fresh install fails

### DISCUSSION THREAD

**Discussion by cardiogramx** [adonisjs/core#3933] [Help] [answered]
I get the following error when I run `yarn` or `npm install`. Any idea on how to resolve this. It started two days ago and happens in docker too. 

> error Command failed.
> Exit code: 128
> Command: git
> Arguments: ls-remote --tags --heads https://github.com/thetutlage/matchit.git
> Directory: /Users/myFolder/Desktop/nodeProject
> Output:
> remote: Repository not found.
> fatal: repository 'https://github.com/thetutlage/matchit.git/' not found
> info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

**Comment by RomainLanz** [comment, marked as answer]
Hey @cardiogramx! 👋🏻 

You must be using an old version. This package was only used in the 5.0 preview.

**Reply by cardiogramx** [reply]
That solves it. Thanks for the help. Whoever needs to upgrade might find [this](https://github.com/adonisjs/core/discussions/2322) helpful.
